apps-rendering/src/components/Standfirst/AnalysisStandfirst.tsx

```diff
@@ -0,0 +1,33 @@
+import { css } from '@emotion/react';
+import { headline } from '@guardian/source-foundations';
+import type { Item } from 'item';
+import { getFormat } from 'item';
+import DefaultStandfirst, { defaultStyles } from './Standfirst.defaults';
+
+interface Props {
+	item: Item;
+}
+
+const analysisStandfirstStyles = css`
```

Comment:
I think we could probably call this `styles` as we're in a clearly named module which indicates Analysis & Standfirst

Reply:
Makes sense, I'll update this :)